tests-functional/tests/reliability/test_community_messages.py

```diff
@@ -0,0 +1,58 @@
+from time import sleep
+from uuid import uuid4
+import pytest
+from tests.test_cases import MessengerTestCase
+from clients.signals import SignalType
+from resources.enums import MessageContentType
+
+
+@pytest.mark.usefixtures("setup_two_nodes")
+@pytest.mark.reliability
+class TestCommunityMessages(MessengerTestCase):
+
+    def test_community_messages_baseline(self, message_count=1, network_condition=None):
```

Comment:
would be great to move baseline as a method into MessengerTestCase as result we will be able to reuse it in functional tests